vcpkg.json

```diff
@@ -47,11 +40,27 @@
     "zlib",
     "pegtl",
     "inja",
-    "ffmpeg",
     "warn",
     "zlib"
   ],
+  "default-features": [
+    "hdf5",
```

Comment:
The HDF5 module is not on by default, so it should not be added here?

Reply:
It used to be on, I will keep it on